app/controller.go

```diff
@@ -165,9 +170,16 @@ func (app *App) commitor() commitor {
 	return func() ResponseCommit {
 
 		// Commit any pending changes.
-		hash, ver := app.Context.balances.Commit()
+		hashb, verb := app.Context.balances.Commit()
```

Comment:
what's verb and verv?

Reply:
version for balance and version for validators